warehouse/scripts/dbt_artifacts/__init__.py

```diff
@@ -193,119 +139,159 @@ def gvattrs(self) -> Dict[str, Any]:
-            DbtMaterializationType.table,
-            DbtMaterializationType.incremental,
-        ):
+        if self.config and self.config.materialized in ("table", "incremental"):
             fillcolor = "aquamarine"
 
         if (
             self.catalog_entry
-            and self.catalog_entry.num_bytes
-            and self.catalog_entry.num_bytes > 100_000_000_000
+            and self.catalog_entry.num_bytes  # type: ignore[attr-defined]
+            and self.catalog_entry.num_bytes > 100_000_000_000  # type: ignore[attr-defined]
             and "clustering_fields" not in self.catalog_entry.stats
             and "partitioning_type" not in self.catalog_entry.stats
         ):
             fillcolor = "red"
 
-        if (
-            self.config.materialized == DbtMaterializationType.view
-            and len(self.children) > 1
-        ):
-            fillcolor = "pink"
+        # TODO: bring me back
+        # if self.config.materialized == "view" and len(self.children) > 1:
+        #     fillcolor = "pink"
 
         return {
             "fillcolor": fillcolor,
         }
 
 
-class TestMetadata(BaseModel):
-    name: str
-    kwargs: Dict[str, Union[str, List, Dict]]
+class RPCNode(BaseRPCNode, NodeModelMixin):
+    pass
+
+
+class SqlNode(BaseSqlNode, NodeModelMixin):
+    pass
+
+
+class GenericTestNode(BaseGenericTestNode, NodeModelMixin):
+    pass
 
 
-class Test(BaseNode):
-    resource_type: Literal[DbtResourceType.test]
-    # test_metadata is optional because singular tests (custom defined) do not have test_metadata attribute
-    # for example: https://github.com/dbt-labs/dbt-docs/blob/main/src/app/services/graph.service.js#L355
-    # ^ singular test is specifically identified by not having the test_metadata attribute
-    test_metadata: Optional[TestMetadata]
+class SnapshotNode(BaseSnapshotNode, NodeModelMixin):
+    pass
+
+
+class SeedNode(BaseSeedNode, NodeModelMixin):
+    @property
+    def gvattrs(self):
+        return {
+            "fillcolor": "green",
+        }
 
 
-Node = Annotated[
-    Union[Seed, Source, Model, Test],
-    Field(discriminator="resource_type"),
+DbtNode = Union[
+    AnalysisNode,
+    SingularTestNode,
+    HookNode,
+    ModelNode,
+    RPCNode,
+    SqlNode,
+    GenericTestNode,
+    SnapshotNode,
+    SeedNode,
 ]
 
 
-class ExposureType(str, Enum):
-    dashboard = "dashboard"
-    notebook = "notebook"
-    analysis = "analysis"
-    ml = "ml"
-    application = "application"
+class SourceDefinition(BaseSourceDefinition, NodeModelMixin):
+    @property
+    def gvattrs(self) -> Dict:
+        return {
+            "fillcolor": "blue",
+        }
+
+    pass
+
+
+class FileFormat(str, Enum):
+    csv = "csv"
+    geojson = "geojson"
+    geojsonl = "geojsonl"
+    json = "json"
+    jsonl = "jsonl"
+
+
+class TileFormat(str, Enum):
+    mbtiles = "mbtiles"
+    pbf = "pbf"
 
 
-class Owner(BaseModel):
-    name: Optional[str]
-    email: str
+# class Column(BaseColumn):
```

Comment:
what is all this commented content?

Reply:
thanks I needed to remove these, not useful anymore